plugin.rb

```diff
@@ -55,6 +57,10 @@
     end
 
     any_user_agent[:disallow] ||= []
-    any_user_agent[:disallow] << "/docs/"
+    any_user_agent[:disallow] << "/#{GlobalSetting.docs_path}/"
+  end
+
+  add_to_serializer(:site, :docs_path, "docs") do
```

Comment:
The third parameter here is `define_include_method = true` by default and I'm not sure passing `"docs"` is helpful. 

Would be good to add a test for this serializer as well.

Reply:
Good catch! I've removed the incorrect `"docs"` and added a serializer test.